recipe/meta.yaml

```
@@ -213,95 +216,84 @@ outputs:
     build:
       number: {{ build_num }}
-      # build cuQuantum Python in a CUDA agnostic way to serve both CUDA 11/12
-      # (note: this would still use cuda_compiler_version to calculate hash)
-      skip: true  # [not (cuda_compiler_version or "").startswith("11") or (py < 39)]
-      script_env:
-        - CUQUANTUM_ROOT=$PREFIX
-        - CUTENSOR_ROOT=$PREFIX
-      script:
-        # The CUDA 11 docker image sets CUDA_PATH for us
-        - export CUDA_PATH=$BUILD_PREFIX/targets/x86_64-linux/   # [linux64 and (cuda_compiler_version or "").startswith("12")]
-        - export CUDA_PATH=$BUILD_PREFIX/targets/sbsa-linux/     # [aarch64 and (cuda_compiler_version or "").startswith("12")]
-        - export CUDA_PATH=$BUILD_PREFIX/targets/ppc64le-linux/  # [ppc64le and (cuda_compiler_version or "").startswith("12")]
-        - cd python
-        # {{ PYTHON }} is not resolved properly in multi-output recipes...
-        - $PREFIX/bin/python -m pip install --no-deps --no-build-isolation -vv .
+      string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ mpi_suffix }}"
+      script: "${RECIPE_DIR}/custatevec/cusv-install"
+      run_exports:
+        - {{ pin_subpackage("custatevec", max_pin="x") }} {{ mpi_prefix }}_*
       ignore_run_exports_from:
-        # let cuquantum handle the CUDA deps
-        - {{ compiler('cuda') }}
+        - {{ compiler("cuda") }}
+        - libcublas  # [cuda_compiler_version in ("12.0", "12.6")]
+      post-link: post-link
     requirements:
       build:
         - {{ stdlib('c') }}
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-        - {{ compiler('cuda') }}
-        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
-        - python >=3.10                       # [build_platform != target_platform]
-        - cython <3                           # [build_platform != target_platform]
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - {{ compiler("cuda") }}
       host:
-        - python >=3.10
-        - pip
-        - cython <3
-        - custatevec  ={{ cusv_version }}
-        - cutensornet ={{ cutn_version }}
-        - packaging
+        - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version in ("11.8", "12.0", "12.6")]
+        - libcublas                                 # [cuda_compiler_version in ("12.0", "12.6")]
       run:
-        - python >=3.10
-        - numpy  >=1.21,<2
-        - cupy   >=13.0
-        - {{ pin_subpackage('custatevec', max_pin='x') }}
-        - {{ pin_subpackage('cutensornet', max_pin='x') }}
-        - cuda-version >=11.0.*,<13.a0  # again, cuQuantum Python is "CUDA agnostic"
+        - cuda-version >={{ cuda_major }}.0,<{{ cuda_major + 1 }}  # [cuda_compiler_version in ("11.8", "12.0", "12.6")]
+        - {{ pin_compatible("cuda-version", max_pin="x", min_pin="x") }}  
+        - libcublas  # [cuda_compiler_version in ("12.0", "12.6")]
+        - cudatoolkit  # [cuda_compiler_version == "11.8"]
```

```suggestion
        - libcublas  # [(cuda_compiler_version or "").startswith("12")]
```
I recommend using this selector instead in all cases because otherwise, it's going to be alot of recipe churn when the the channel migrates to the next 12.x release.